ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -22,40 +22,268 @@ resources:
+      PODS_PER_NODE: "100"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
       REPLICAS: "24"
+      ZONES_COUNT: "3"
     test:
-    - ref: openshift-qe-cluster-density-v2
+    - ref: openshift-qe-node-density-heavy
+    workflow: rosa-aws-sts-hypershift
+- as: ocp-qe-perfscale-ci-tests-rosa-network-perf
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      REPLICAS: "3"
+    test:
+    - chain: openshift-qe-network-perf
     workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-network-perf-24nodes
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-network-perf
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-hcp-network-perf-24nodes
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
+      REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-network-perf
+    workflow: rosa-aws-sts-hypershift
+- as: ocp-qe-perfscale-ci-tests-rosa-router-perf
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      REPLICAS: "3"
+    test:
+    - ref: openshift-qe-router-perf
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-router-perf-24nodes
+  cron: 0 12 * * 0
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - ref: openshift-qe-router-perf
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-hcp-router-perf-24nodes
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
+      REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - ref: openshift-qe-router-perf
+    workflow: rosa-aws-sts-hypershift
```

Comment:
for 4.12 we separated rosa and rosa hypershift. should we do the same thing here?

Reply:
That was separated for prod vs stage. Not so much rosa vs rosa hypershift.